pkg/cri/server/image_pull.go

```diff
@@ -194,8 +209,14 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 		}
 	}
 
-	log.G(ctx).Debugf("Pulled image %q with image id %q, repo tag %q, repo digest %q", imageRef, imageID,
-		repoTag, repoDigest)
+	imageNamed, _ := distribution.ParseNamed(ref)
+	domain := distribution.Domain(imageNamed)
+	size, _ := image.Size(ctx)
+	imagePullingSpeed := float64(size) / time.Since(startTime).Seconds()
+	imagePullThroughput.WithLabelValues(domain).Observe(imagePullingSpeed)
+
+	log.G(ctx).Infof("Pulled image %q with image id %q, repo tag %q, repo digest %q, size %q in %v s", imageRef, imageID,
+		repoTag, repoDigest, strconv.FormatInt(size, 10), time.Since(startTime).Seconds())
```

Comment:
I think the latency metric is import, although it's not comparable for different images, we can add the image ref as the image label, when building dashboard, we can monitor the latency for each images. This has turned out to be a very useful metrics in our operations.

Reply:
I added a label before, but containerd reviewer/approver think this will make too many metrics for this.

See https://github.com/containerd/containerd/pull/7313#discussion_r983983740. 

> I don't think it's reasonable to assume that containerd will only pull from 10 distinct domains during its lifetime (which appears to be the actual distinction, not registries?). Some container registry providers use many distinct domains (for example, Amazon ECR has a different domain name for each customer account).